components/prism-liquid.js

```diff
@@ -1,8 +1,27 @@
-Prism.languages.liquid = {
+Prism.languages.liquid = {};
+Object.assign(Prism.languages.liquid, {
```

Comment:
In the lines below there’s a spot where we’re referring to the Prism.languages.liquid object in an `inside` block. How would you suggest getting around this?

Reply:
I'm not sure how we deal with it elsewhere, but a getter would work.

Reply:
We typically do it like [this](https://github.com/PrismJS/prism/blob/82d0ca15e85574e14d5714cdf2b989a5a9a36143/components/prism-brightscript.js#L18).

I also want to mention #2948. This would allow you to write `inside: 'liquid'`.